pyproject.toml

```
@@ -120,6 +120,7 @@ addopts = ["-ra", "--strict-markers", "--strict-config", "--showlocals"]
 log_cli_level = "INFO"
 xfail_strict = true
 filterwarnings = ["error",
+    "ignore::DeprecationWarning: pkg_resources is deprecated as an API"
```

```suggestion
    "ignore::DeprecationWarning:pkg_resources.*"
```
that should fix it properly